### PR TITLE
CSCQVAIN-320: Add schema scripts, update schema

### PR DESCRIPTION
**Pull request by tahme**

### CODE REVIEW

File: scripts/schema/compare_schemas.js [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+	if (deep === undefined) {
+		deep = false
+	}
+
+	const remtree = (v, path) => {
+		if (v === undefined) {
+			return
+		}
+		if (!deep) {
+			diffs.push({ path: path, change: "-", val: v })
```
Comment (Kauhia) [inline scripts/schema/compare_schemas.js:19]: Why is this put to diffs ?

Comment (Kauhia) [reply scripts/schema/compare_schemas.js:19]: And why is deep needed?

File: scripts/schema/compare_schemas.js [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+				diffs.push({ path: path, change: "-", val: v })
+			}
+		}
+	}
+
+	const addtree = (v, path) => {
+		if (v === undefined) {
+			return
+		}
+		if (!deep) {
```
Comment (Kauhia) [inline scripts/schema/compare_schemas.js:36]: deep?

Comment (Kauhia) [reply scripts/schema/compare_schemas.js:36]: Ahh, are those two functions otherwise similar but other adds and the other removes?

**Review by Kauhia — CHANGES_REQUESTED** [review]
Did you check that license url and dropdown work after these changes?

File: scripts/schema/compare_schemas.js (hunk 1 above)
Comment (tahme) [reply scripts/schema/compare_schemas.js:19]: Deep was useful for debugging but the output tends to be verbose. For example, if you have added `{a: {b: 1, c: { d:2, e: 3 }}`, you get output like (when deep is disabled):
```
+ a: [object]
```
with deep enabled you get
```
+ a.b: 1
+ a.c.d: 2
+ a.c.e: 3
```

File: scripts/schema/compare_schemas.js (hunk 2 above)
Comment (tahme) [reply scripts/schema/compare_schemas.js:36]: Yes. I guess I could have just added +/- as a parameter instead of having two functions.

**Comment by tahme** [comment]
> Did you check that license url and dropdown work after these changes?

Seemed to work the same way as before.

**Review by Kauhia — APPROVED** [review]

**Pull request merged**